l10n_ch_account_statement_base_import/statement.py

```diff
@@ -26,6 +26,15 @@
 import tarfile
 import base64
 from datetime import date
+import hashlib
+
+
+class AccountBankStatement(orm.Model):
+    _inherit = "account.bank.statement"
+
+    _columns = {
+        'checksum': fields.char(_('Checksum')),
```

Comment:
No need to specify string as translatable. All field descriptions are already translatable